opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -97,14 +115,6 @@ message ScopeMetrics {
 // - DataPoint contains timestamps, attributes, and one of the possible value type
 //   fields.
 //
-//     Metric
-//  +------------+
-//  |name        |
-//  |description |
-//  |unit        |     +------------------------------------+
-//  |data        |---> |Gauge, Sum, Histogram, Summary, ... |
-//  +------------+     +------------------------------------+
-//
```

Comment:
This doesn't need to be removed. It contextualizes the following diagram about metric data.

Reply:
I removed that based on @bogdandrutu's advice: https://github.com/open-telemetry/opentelemetry-proto/pull/519#discussion_r1452724379

Reply:
His comment was made when this PR was changing the documentation of this declaration. It doesn't seem relevant to me given this PR has moved the added documentation elsewhere.

Reply:
Alright, I added it back.